QgisModelBaker/tests/test_import.py

```diff
@@ -85,7 +85,7 @@ def test_import_postgis(self):
             """.format(importer.configuration.dbschema))
         record = next(cursor)
         self.assertIsNotNone(record)
-        self.assertEqual(record[0], 38)  # t_id for 'Unidad_Derecho'
+        self.assertEqual(record[0], 6)  # t_id for 'Unidad_Derecho'
```

Comment:
Is it meaningful to test these ids?

Reply:
Not that much. We can remove them in an coming commit.